tests/import_source/import_source.cpp

```diff
@@ -309,3 +309,37 @@ TEST(ImportSource, createLinkedMultiple)
     EXPECT_FALSE(c1->isImport());
     EXPECT_FALSE(u1->isImport());
 }
+
+TEST(ImportSource, addImportComponentBeforeAddingToModel)
```

Comment:
Shouldn't we also have a similar test for adding `c2` to `c1` **after** adding `c1` to the model?

Reply:
Addressed in [3b34378].(https://github.com/cellml/libcellml/pull/791/commits/3b34378682054de9687f67f74a3b5b4eb37db118)